src/firebase.ts

```diff
@@ -0,0 +1,37 @@
+import { initializeApp } from 'firebase/app';
+import type { User } from 'firebase/auth';
+import {
+	getAuth,
+	createUserWithEmailAndPassword,
+	signInWithEmailAndPassword,
+	signOut as authSignOut,
+	onAuthStateChanged
+} from 'firebase/auth';
+
+// Initialize Firebase
+initializeApp({
```

Comment:
Tyhle údaje bychom pak asi měli přesunout do .env souboru, už jenom proto, že repozitář je veřejný.

Reply:
Určitě jsem za! Říkal jsem si, že to možná budeš chtít nalinkovat na svůj firebase a na svůj PlanetScale, když jsi i owner repa. :)

Reply:
To je asi dobrý nápad, aby to bylo pod jednou osobou, můžeme to pořešit později, až budeme linkovat db